githook/prepare-commit-msg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import sys
+
+def is_integer(value: str) -> bool:
+    try:
+        int(value)
+        return True
+    except ValueError as _:
+        pass
+
+    return False
+
+def check_commit_message() -> int:
+    commit_message_filename: str = sys.argv[1]
+    with open(commit_message_filename) as file:
+        first_line: str = file.readline().strip()
+
+    issue_name = first_line.split(" ")[0]
+
+    # Allow fixup commits
+    if issue_name == "fixup!":
```

Comment:
What about my favourite ~ ? 🙂 Would be nice to allow it as well

Reply:
Sure :) will add